README.md

```diff
@@ -1,2 +1,200 @@
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]** Configured linters for react app
+- **[key_feature_2]** Basic Setup of React
+- **[key_feature_3]** Professional readme file 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Will be added later](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```  
+  Node
+   VS Code or any other editor
+   
+```
+ 
+
+### Setup
+Clone this repository to your desired folder:
+
+```
+  sh
+  cd my-project
+   git clone https://github.com/sajibarafatsiddiqui/Math-Magicians.git   
+
+```
+
+
+### Install
+
+Install this project with:
+
+```
+   sh
+   cd my-project
+   npm init -y
+   npm install
+   
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+   sh
+   cd math-magicians
+   npm start
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Sajib Arafat Siddiqui**
+
+- GitHub: [@sajibarafatsiddiqui](https://github.com/sajibarafatsiddiqui)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**will add database
+- [ ] **[new_feature_2]**will add backend server
+- [ ] **[new_feature_3]**will make some test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my morning session teammates who gave me suggestions regarding this project. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
**[Optional]**

- [ ] Kindly remove this **NOTE** because this is just an instruction for you to make an LICENSE file which you already have so kindly remove this **NOTE** to make your readme file descriptive and professional :blush: